eml/api/userApi.js

```diff
@@ -109,18 +115,24 @@ export const enrollInCourse = async (user_Id, course_Id) => {
   try {
     // When user enrolls in a course it sends the course id to the database,
     // and then stores the course and completion status in the user document.
-    const res = await axios.post(url + '/api/eml/' + user_Id + '/enroll/' + course_Id);
+    const res = await axios.post(
+      url + "/api/eml/" + user_Id + "/enroll/" + course_Id
+    );
 
     // First time user enrolls in course
     if (!res.data.course) {
-      return res.data
+      return res.data;
     }
 
     // if the course already exists, return the completion status of the course, sections, and exercises
     return {
       courseCompletion: res.data.course.isComplete,
-      sectionCompletion: res.data.course.sections.map(section => section.isComplete),
-      exerciseCompletion: res.data.course.sections.map(section => section.exercises.map(exercise => exercise.isComplete))
+      sectionCompletion: res.data.course.sections.map(
+        (section) => section.isComplete
+      ),
+      exerciseCompletion: res.data.course.sections.map((section) =>
+        section.exercises.map((exercise) => exercise.isComplete)
+      ),
     };
```

Comment:
UpdateFirstname, updateLastname, UpdateEmail can be deleted. --> Ask @Mafusn if any questions